packages/cli/src/services/check-parser/package-files/package-manager.ts

```
@@ -0,0 +1,384 @@
+
+  // Finally, try to find a relevant executable.
+  //
+  // This can generate a whole bunch of path lookups. Try one by one despite
+  // async support.
+  const lookup = new PathLookup()
+  for (const detector of detectors) {
+    try {
+      await detector.detectExecutable(lookup)
+      return detector
+    } catch {
+      continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)